test/function-debounce/index.js

```diff
@@ -0,0 +1,19 @@
+var test = require('tape');
+var debounce = require('../../packages/function-debounce');
+
+test('will not invoke until it has stopped being called for N milliseconds', function (t) {
```

Comment:
Added test and it failed =P. Added an escape conditional to invoke immediately if wait time passed is equal to 0.